src/simplification/SimplifyTypes.ts

```diff
@@ -0,0 +1,109 @@
+import { Schema } from "models/Schema";
+
+
+/**
+ * Find the types for a simplified version of a schema
+ * 
+ * @param schema to find the simplified types for
+ */
+export default function simplifyTypes(schema: Schema | boolean) : string[] {
+  //If we find absence of data format ensure all types are returned
+  if(typeof schema === "boolean"){
```

Comment:
`type: false` should be ignored? In this case we go with `true` and `false` and I think that we should go only with `true`. Am I correct?

Reply:
Hmm, I decided to include both here since if you have 
```
{
  "properties": { 
    "testProp1": false,
    ...
  },
  ...
}
```
I would still like to have the property there. It might be a false assumption, but for now I think we will keep it as is 😄

Reply:
I have problem with `false`. I remember that we had a talk about it, but `true` is understandable, it means every type, but `false` should be in my opinion undefined :)

Reply:
So you would expect that object above to not contain the property at all?

Reply:
I can't figure what `false` can mean here tbh 😄

Reply:
In validation it is simply "reject all values". Hmm I guess I answered my own question that it should be ignored 🤔

Reply:
Yeah, if a schema is `false` it means it would always evaluate to fail to validate, so I wonder who's actually using this. I'd just parse `true` as `any` and in case of `false` I'd raise an error maybe? Not sure.

Reply:
We could do that as well yea.

Reply:
Lets do that for now then 😄